src/features/rejection/add-question/add-question-hoc.js

```diff
@@ -0,0 +1,30 @@
+import { connect } from 'react-redux';
```

Comment:
I usually call the container component `index.js` inside the component/feature folder. `add-question-container.js` is also OK.

Reply:
Renamed to `index.js`.